templates/partials/footer.html

```diff
@@ -14,8 +14,8 @@ <h5 class="mzp-c-footer-heading">
                         <li><a href="https://blog.mozilla.org/press/">Press Center</a></li>
                         <li><a href="https://blog.mozilla.org/">Corporate Blog</a></li>
 
-                        <li><a href="/en-US/careers/" data-link-type="footer" data-link-name="Careers">Careers</a></li>
-                        <li><a href="/en-US/contact/" data-link-type="footer" data-link-name="Contact">Contact</a></li>
+                        <li><a href="https://mozilla.org/en-US/careers/" data-link-type="footer" data-link-name="Careers">Careers</a></li>
```

Comment:
We should add `www` and probably omit the `en-US` (even though MIECO isn't localized, we can still let bedrock's locale detection do its thing)

Reply:
Good idea